### PR TITLE
feat(client): support auto copy artifact when run or serve

### PR DESCRIPTION
**Pull request by goldenxinxing**
## Description

1. Model run and serve's uriArgs can pass remoteUri, such as model-uri: cloud://server/project/simple-test/version/123456, runtime-uri: cloud://server/project/simple-test/version/654321
2. entrypoint use the unify cmd

## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [x] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [x] run code format and lint check
- [x] add unit test
- [ ] add necessary doc

### CODE REVIEW

**Review by jialeicui — DISMISSED** [review]
LGTM

File: client/scripts/sw-docker-entrypoint [hunk 1]
```diff
@@ -36,53 +38,49 @@ welcome() {
...
         $(bash "${SW_USER_RUNTIME_WORKDIR:-/opt/starwhale.user/runtime}"/activate.sw)
     fi
 
-    if [ "${SW_INSTANCE_URI}" != "local" ]
-    then
-        # only remote
-        echo '-->[Preparing] pulling model ...'
+    if [ "${SW_INSTANCE_URI}" == "local" ]; then
+        if [ -z "${SW_MODEL_URI}" ]; then
+            SW_MODEL_URI="local/project/${SW_PROJECT}/model/${SW_MODEL_VERSION}"
```
Comment (tianweidut) [inline client/scripts/sw-docker-entrypoint:47]: miss name?

File: client/starwhale/core/dataset/tabular.py [hunk 2]
```diff
@@ -584,7 +585,7 @@ def __init__(
         self.session_end = session_end
         self.dataset_uri = dataset_uri
         self.run_env = _RunEnvType.POD
-        self.consumer_id = os.environ.get(ENV_POD_NAME)
+        self.consumer_id = os.environ.get(ENV_POD_NAME) or uuid1().hex
```
Comment (tianweidut) [inline client/starwhale/core/dataset/tabular.py:588]: replace `uuid1().hex` with `starwhale.utils.gen_uniq_version` ?

Comment (goldenxinxing) [reply client/starwhale/core/dataset/tabular.py]: done

**Review by tianweidut — APPROVED** [review]

**Pull request merged**